test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/BookkeeperK8sService.java

```diff
@@ -81,6 +81,22 @@ public boolean isRunning() {
                                               t -> new TestFrameworkException(RequestFailed, "Failed to fetch ServiceDetails for bookkeeper", t));
     }
 
+    protected Map<String, Object> patchSpecBuilder(int newInstanceCount) {
+
+        final Map<String, Object> bookkeeperSpec = ImmutableMap.<String, Object>builder()
+                .put("replicas", newInstanceCount)
+                .build();
+
+        return ImmutableMap.<String, Object>builder()
```

Comment:
Create a new utility method (Utils.java) that takes pravegaSpec as parameter and adds the rest of the values in this code. It can be invoked here as well as in ln #173

Reply:
done